zubhub_frontend/zubhub/src/assets/js/styles/components/hamburger_menu/hamburgerSidebarStyles.js

```diff
@@ -0,0 +1,72 @@
+const styles = theme => ({
+  hamburgerSidebarStyle: {
+    '& .MuiDrawer-paper': {
+      height: '85%',
+      width: '70%',
+      top: '3.5em',
+      left: '30%',
+      borderRadius: '10px',
+    },
+  },
+
+  textDecorationNone: {
+    textDecoration: 'none',
+  },
+
+  logOutStyle: {
+    borderTop: '1px solid #C4C4C4',
+    backgroundColor: '#ededed',
+    height: '10%',
+    width: '100%',
+    paddingTop: '2%',
+    paddingBottom: '2%',
+    paddingLeft: '3%',
+    paddingRight: '3%',
+  },
+
+  menuItemSelected: {
+    backgroundColor: 'black',
+  },
+
+  menuItemStyle: {
+    backgroundColor: '#ededed',
+    height: '10%',
+    width: '100%',
+    paddingTop: '2%',
+    paddingBottom: '2%',
+    paddingLeft: '3%',
+    paddingRight: '3%',
```

Comment:
I don't think this is necessary. One thing you could try is centering the daughter div and letting that handle its own styling. Same applies to ```logOutStyle```